samples/esb/prx/src/main.c

```diff
@@ -107,27 +105,29 @@ void event_handler(struct esb_evt const *event)
 int clocks_start(void)
 {
 	int err;
-	const struct device *clk;
-
-	clk = device_get_binding(DT_INST_LABEL(0));
-	if (!clk) {
-		LOG_ERR("Clock device not found!");
-		return -EIO;
-	}
+	int res;
+	struct onoff_manager *clk_mgr;
+	struct onoff_client clk_cli;
 
-	err = clock_control_on(clk, CLOCK_CONTROL_NRF_SUBSYS_HF);
-	if (err && (err != -EINPROGRESS)) {
-		LOG_ERR("HF clock start fail: %d", err);
-		return err;
+	clk_mgr = z_nrf_clock_control_get_onoff(CLOCK_CONTROL_NRF_SUBSYS_HF);
```

Comment:
Isn't z_ functions internal functions?

This doesn't look proper to me.

Reply:
As far as I know, there is no other way of accessing

Reply:
Maybe there should be one? It looks hacky this way.  @anangl?